ios/GoogleService-Info.plist

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
this file should not be on Github ! Please rdelete this file from git with "git rm ios/GoogleService-Info.plist"